policyengine_us/variables/household/demographic/person/ssn_card_type.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class SSNCardType(Enum):
+    CITIZEN = "Citizen"
+    NON_CITIZEN_VALID_EAD = "Non-citizen with valid EAD card"
+    OTHER_NON_CITIZEN = "Other non-citizen"
+
```

Comment:
Can you add "NONE" type - just spoke with @vahid-ahmadi and that is part of the data